inst/htmljs/animint.js

```diff
@@ -1709,11 +1709,8 @@ var animint = function (to_select, json_file) {
 	  return d["clickSelects.variable"] + " " + d["clickSelects.value"];
 	};
       }
-      // If elements have <title>, remove it
-      if(elements.select("title")[0][0]!== null){
-          elements.selectAll("title")
-            .remove()
-      }
+      // if elements have an existing title, remove it.
+      elements.selectAll("title").remove();
```

Comment:
@tdhock Not really sure why this was happening. You could try running [this test](https://github.com/tdhock/animint/blob/master/tests/testthat/test-tt-inter.R) locally to check for tooltips. It is a bit more extensive than the one we have for wercker/Travis.